### PR TITLE
added info to non forgable txs

### PR DESCRIPTION
**Pull request by Mikelle**
Closes https://github.com/hermeznetwork/hermez-node/issues/726

### What does this PR does?

Added info to nonforgeable txs and deleted checker for enough balance in splitL2ForgableAndNonForgable, according to the last comment in the task 

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Include tests
- [x] Respect code style and lint
- [ ] Update swagger file (if needed)
- [ ] Update documentation (*.md) (if needed)

### CODE REVIEW

File: txselector/txselector_test.go [hunk 1]
```diff
@@ -411,123 +411,6 @@ func TestGetL2TxSelectionMinimumFlow0(t *testing.T) {
 	txsel.LocalAccountsDB().Close()
 }
 
-func TestPoolL2TxsWithoutEnoughBalance(t *testing.T) {
```
Comment (Mikelle) [inline txselector/txselector_test.go:414]: deleted this test, because we don't check the balance in splitL2ForgableAndNonForgable

**Review by jeffprestes — COMMENTED** [review]
Before to send the PR, have you tested it in internaltestnet environment with a node running as a coordinator?

**Review by Pantani — APPROVED** [review]

File: txselector/txselector.go [hunk 2]
```diff
@@ -762,17 +760,15 @@ func splitL2ForgableAndNonForgable(tp *txprocessor.TxProcessor,
 	var l2TxsForgable, l2TxsNonForgable []common.PoolL2Tx
 	for i := 0; i < len(l2Txs); i++ {
 		accSender, err := tp.StateDB().GetAccount(l2Txs[i].FromIdx)
+		l2Txs[i].Info = fmt.Sprintf("Tx not selected because failed to get account with idx %d", l2Txs[i].FromIdx)
```
Comment (Pantani) [inline txselector/txselector.go:763]: You must set this message inside the error condition below

Comment (Pantani) [reply txselector/txselector.go:763]: Maybe it is better to add the parameter name that was wrong (`FromIdx`) into the message:
```suggestion
		l2Txs[i].Info = fmt.Sprintf("Invalid transaction, FromIdx account not found %d", l2Txs[i].FromIdx)
```

**Review by Pantani — CHANGES_REQUESTED** [review]

File: txselector/txselector.go [hunk 3]
```diff
@@ -762,17 +760,15 @@ func splitL2ForgableAndNonForgable(tp *txprocessor.TxProcessor,
 	var l2TxsForgable, l2TxsNonForgable []common.PoolL2Tx
 	for i := 0; i < len(l2Txs); i++ {
 		accSender, err := tp.StateDB().GetAccount(l2Txs[i].FromIdx)
+		l2Txs[i].Info = fmt.Sprintf("Invalid transaction, FromIdx account not found %d", l2Txs[i].FromIdx)
```
Comment (Pantani) [inline txselector/txselector.go:763]: You must set this message inside the error condition below. Now it's outside the error check:

```go
		accSender, err := tp.StateDB().GetAccount(l2Txs[i].FromIdx)
		if err != nil {
			l2Txs[i].Info = fmt.Sprintf("Invalid transaction, FromIdx account not found %d", l2Txs[i].FromIdx)
			l2TxsNonForgable = append(l2TxsNonForgable, l2Txs[i])
			continue
		}
```

Comment (Mikelle) [reply txselector/txselector.go:763]: Sorry, the commit was not added to the push.

**Review by Pantani — APPROVED** [review]

**Pull request merged**